### PR TITLE
add metricbeat

### PR DESCRIPTION
**Pull request by SergiiGlad**

### CODE REVIEW

File: Terraform/GCP/k8s/values/elastic.yaml [hunk 1]
```diff
@@ -217,29 +204,256 @@ vector:
...
-          action: create
-          index: vector-logs-k3s-{{`{{ kubernetes.pod_name }}`}}-%Y-%m-%d
+          action: index
+          index: vector-logs-kubernetes-%F
         auth:
           strategy: basic
           user: "${ES_USERNAME}"
           password: "${ES_PASSWORD}"
+
+metricbeat:
```
Comment (DmyMi) [inline Terraform/GCP/k8s/values/elastic.yaml:233]: need to change this to
```yaml
metricbeat:
  enabled: true
  daemonset:
    # Include the daemonset
    enabled: true
  # other config
```

to be able to disable/enable this subchart

File: k8s/elastic/Chart.yaml [hunk 2]
```diff
@@ -20,3 +20,8 @@ dependencies:
     version: 0.20.1
     condition: vector.enabled
     repository: https://helm.vector.dev
+  - name: metricbeat
+    version: 8.5.1
```
Comment (DmyMi) [inline k8s/elastic/Chart.yaml:24]: change version to `7.17.3` to be consistent with the other parts of Elastic stack. Not sure we are ready to move to 8.x elastic

**Review by DmyMi — APPROVED** [review]

File: Terraform/GCP/k8s/values/elastic.yaml (hunk 1 above)
Comment (SergiiGlad) [reply Terraform/GCP/k8s/values/elastic.yaml:235]: fixed

File: k8s/elastic/Chart.yaml (hunk 2 above)
Comment (SergiiGlad) [reply k8s/elastic/Chart.yaml]: fixed

**Review by DmyMi — APPROVED** [review]

**Pull request merged**